app/helpers/physical_server_helper/textual_summary.rb

```diff
@@ -109,7 +109,18 @@ def textual_ipv4
     # It is possible for guest devices not to have network data (or a network
     # hash). As a result, we need to exclude guest devices that don't have
     # network data to prevent a nil class error from occurring.
-    {:label =>  _("IPv4 Address"), :value => @record.hardware.guest_devices.reject { |device| device.network.nil? }.collect { |device| device.network.ipaddress }.join(", ") }
+    devices_with_networks = @record.hardware.guest_devices.reject { |device| device.network.nil? }
+    ip_addresses = devices_with_networks.collect { |device| device.network.ipaddress }
+
+    # It is possible that each network entry can have multiple IP addresses, separated
+    # by commas, so first convert the array into a string, separating each array element
+    # with a comma. Then split this string back into an array using a comma, possibly
+    # followed by whitespace as the delimiter. Finally, iterate through the array
+    # and convert each element into a URL containing an IP address.
+    ip_addresses = ip_addresses.join(",").split(/,\s*/)
```

Comment:
@skovic Can you write a spec for `textual_ipv4` for some of these IP address related use cases (comma separated multiple IP addresses, single IP addresses etc) ?

Reply:
@AparnaKarve As requested, I wrote a spec test for textual_ipv4. Thanks

Reply:
@skovic Thanks - the spec looks good!